src/components/Footer/Footer.tsx

```diff
@@ -1,34 +1,52 @@
 export default function Footer() {
   return (
-    <footer
-      className={'flex h-24 w-full items-center justify-start bg-black text-inverted sm:flex-row'}>
-      <div
-        className={' ml-6 text-lg sm:mx-36 sm:flex sm:items-center sm:gap-36'}
-      >
-        <ul
-          className={
-            'flex flex-wrap gap-x-3 sm:order-2 sm:flex-col sm:justify-center'
-          }
-        >
-          <li>
+    <footer className={`flex h-28 w-full items-center justify-start bg-black text-inverted`}>
+      <div className={`sm:text-lg ml-6 text-base sm:ml-36 sm:flex sm:items-center sm:gap-28`}>
+        <div className={`flex gap-x-3 sm:order-2 flex-col sm:justify-center mb-3`}>
+          <div>
             <a href="tel:(555) 123-45675" rel="noopener noreferrer">
```

Comment:
why not add a target_blank attribute to the 'a' tag so that when we click on the link we do not leave our application but open a new tab?

Reply:
fixed